tests/framework/extensions/charts/gatekeeperresources.go

```diff
@@ -0,0 +1,47 @@
+package charts
+
```

Comment:
Could you add comments here as well? For all exported structs/functions please add comments so they are generated in our godocs

Reply:
added comment